proto/lbm/token/v1/event.proto

```diff
@@ -7,40 +7,48 @@ import "lbm/token/v1/token.proto";
 
 option go_package = "github.com/line/lbm-sdk/x/token";
 
+// Deprecated: use typed events.
```

Comment:
What does this mean? will These `EventType` enum be deprecated?

Reply:
Yes, I meant to deprecate it. `EventType` enum is for the legacy events.
However, the deprecation no longer affects anything, because we will not update the module.